pyvrp/cpp/search/Route.h

```diff
@@ -18,8 +18,10 @@ class Route
     ProblemData const &data;
     size_t const vehicleType_;
 
-    std::vector<Node *> nodes;  // List of nodes (in order) in this solution.
-    CircleSector sector;        // Circle sector of the route's clients
+    // List of nodes (in order) in this solution. Start depot is not included;
+    // last element is the endDepot. See LS for details.
+    std::vector<Node *> nodes;
```

Comment:
Question: What's the reason here again behind the start depot not being included but the end depot is included? Why can we not remove the last depot?

Reply:
I don't know? I wanted to get rid of that anyway, so let me try to remove that now.

Reply:
Tried doing this. It's surprisingly complicated, because many things assume we can go to the end depot (e.g., `nodes.back()` assumes nodes is non-empty, the `Route::update()` mechanism needs to handle the end depot as well). I think it's best to leave this for now.